attributes/default.rb

```diff
@@ -18,4 +18,7 @@
 #
 
 default['build-essential']['compile_time'] = false
-default['build-essential']['msys']['path'] = "#{ENV['SYSTEMDRIVE']}\\msys"
+default['build-essential']['mingw']['prefix'] = "#{ENV['SYSTEMDRIVE']}\\mingw"
```

Comment:
Instead of `node['build-essential']['mingw']['prefix']` should just expose two separate attributes (open to better names) like:

```
default['build-essential']['mingw']['32_path'] = "#{ENV['SYSTEMDRIVE']}\\mingw\\tool32"
default['build-essential']['mingw']['64_path'] = "#{ENV['SYSTEMDRIVE']}\\mingw\\tool64"
```

This will save having to compute the paths on the fly in other cookbooks. For example compare these two implementation snippets of code (taken from my WIP branch of https://github.com/chef-cookbooks/omnibus):

```ruby
mingw_path = if windows_arch_i386?
  ::File.join(node['build-essential']['mingw']['prefix'], 'tool32')
else
  ::File.join(node['build-essential']['mingw']['prefix'], 'tool64')
end
```

```ruby
mingw_path = if windows_arch_i386?
  node['build-essential']['mingw']['32_path']
else
  node['build-essential']['mingw']['64_path']
end
```